api/python/cellxgene_census/pyproject.toml

```diff
@@ -31,7 +31,8 @@ dependencies= [
     # NOTE: the tiledbsoma version must be >= to the version used in the Census builder, to
     # ensure that the assets are readable (tiledbsoma supports backward compatible reading).
     # Make sure this version does not fall behind the builder's tiledbsoma version.
-    "tiledbsoma~=1.8.0",
+    # TODO: When ready to release 1.12.0 update tiledbsoma version to 1.9.x and delete this temporary comment 
```

Comment:
time to delete the comment?

BTW, I believe `~=1.9.1` means anything >1.9.1 and <1.10.0, so this pin should be fine (1.9.0 never existed in the wild)

Reply:
will remove comment.

> I believe ~=1.9.1

Yea that is my understanding too